### ISSUE TITLE
Hosted MockBot speech service samples don't work on Chrome 80

### ISSUE THREAD

**Issue by jbgh2** [microsoft/BotFramework-WebChat#2951]
## Version

Following the link to the hosted example on GitHub.

## Describe the bug

The hosted MockBot example links in examples b,c and d don't work on Chrome 80 on Windows 10.

Clicking on the microphone icon causes it to briefly flash but no speech is detected.

## Steps to reproduce

1. Open Chrome 80 on Windows 10
2. Follow link to MockBot: https://microsoft.github.io/BotFramework-WebChat/03.speech/b.cognitive-speech-services-js
3. Press microhpone button
4. Talk
5. Observe that nothing happens

## Expected behavior

Sample should be able to listen and respond to voice commands.

## Additional context

My version of Chrome is completely up to date and has no extensions installed.
Same problem is observed on pre-chromium Edge.
The direct line sample works as expected.

[Bug]

**Comment by compulim** [comment]
Thanks, will take a look at that.

I loaded Web Chat 4.7.1 on a page outside of the sample and it works. I am guessing something on the page itself.

**Comment by compulim** [comment]
Quick fix, line 83 of https://github.com/microsoft/BotFramework-WebChat/blob/master/samples/03.speech/b.cognitive-speech-services-js/index.html#L83.

```diff
  const webSpeechPonyfillFactory = await window.WebChat.createCognitiveServicesSpeechServicesPonyfillFactory({
    // We are passing the Promise function to the "credentials" field.
    // This function will be called every time the token is being used.
    credentials: fetchSpeechServicesCredentials,

    // The following code is needed only for Web Chat < 4.8.
    // Starting from 4.8, we will support the newer "credentials" option, which is preferred over "authorizationToken" and "region".
    authorizationToken: () =>
      fetchSpeechServicesCredentials().then(({ authorizationToken }) => authorizationToken),
-   region: fetchSpeechServicesCredentials().then(({ region }) => region)
+   region: 'westus'
  });
```

tl;dr, `region` must be a `string`, cannot be a `Promise`.

**Comment by compulim** [comment]
I tested https://microsoft.github.io/BotFramework-WebChat/03.speech/b.cognitive-speech-services-js on Chrome 80 and it is fixed.

It is fixed as we released Web Chat 4.8.0, the sample code works properly.
